holoviews/element/raster.py

```diff
@@ -408,45 +211,157 @@ class Image(SheetCoordinateSystem, Raster):
     access to the data, via the .data attribute.
     """
 
-    bounds = param.ClassSelector(class_=BoundingRegion, default=BoundingBox(), doc="""
+    bounds = param.ClassSelector(class_=BoundingRegion, default=None, doc="""
```

Comment:
Though why was it moved to the constructor? Isn't this still a sensible default?